packages/eslint-config/index.js

```diff
@@ -14,7 +14,12 @@ module.exports = {
     'react-refresh',
   ],
   rules: {
-    'prettier/prettier': ['error'],
+    'prettier/prettier': [
+      'error',
+      {
+        endOfLine: 'auto',
+      },
```

Comment:
We shouldn't be changing the `endOfLine` default, we want to keep it to `lf` instead of `crlf`. Might need to configure something on Windows so that line endings are handled correctly in git.